db.js

```
@@ -72,7 +72,7 @@ exports.init = function (sbot, config) {
   const state = {}
 
   sbot.close.hook(function (fn, args) {
-    close(() => {
+    close((err) => {
```

This change does nothing right?

---

I'm okay with leaving it in either case, just mostly trying to understand

---

Yep, does nothing but I thought it was good to put there just to signal that we are ignoring the errors.

---

Great, thanks